go/obscuronode/enclave/rawdb/accessors_indexes.go

```diff
@@ -0,0 +1,154 @@
+package rawdb
+
+import (
+	"bytes"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethdb"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/obscuronet/obscuro-playground/go/log"
+	"github.com/obscuronet/obscuro-playground/go/obscuronode/enclave/core"
+)
+
+// ReadTxLookupEntry retrieves the positional metadata associated with a transaction
```

Comment:
@tudor-malene I think we should make it clear using a comment in cases where we've copied functions directly from Geth.

Reply:
ok. I'll add a readme in the package. The entire thing is a slightly modified copy of the geth stuff